interface/super/edit_list.php

```diff
@@ -1002,8 +1002,14 @@ function mysubmit() {
     }
   }
   else {
-    $res = sqlStatement("SELECT * FROM list_options WHERE " .
-      "list_id = '$list_id' ORDER BY seq,title");
+    /*
+     *  Add edit options to show or hide in list management
+     */
+    $res = sqlStatement("SELECT lo.* 
+                         FROM list_options as lo 
+                         right join list_options as lo2 on lo2.option_id = lo.list_id AND lo2.edit_options = 1
+                         WHERE lo.list_id = '{$list_id}' AND lo.edit_options = 1
+                         ORDER BY seq,title");
```

Comment:
awesome.
I get it now; you are having the main list entry decide whether to show the list (and not allow the hiding of inidividual entries in the list; ie. it is all or nothing).
I'm going to bring this in and then add your explanation as a comment.